WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-ColHeader {
+	border-bottom:0px;
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskListViewItem@
```

Comment:
Property names should indicate that it is for preview pane right and not for preview pane bottom